sdk/playwrighttesting/microsoft-playwright-testing/src/common/playwrightServiceConfig.ts

```diff
@@ -26,9 +25,14 @@ class PlaywrightServiceConfig {
     if (options?.exposeNetwork) {
       this.exposeNetwork = options.exposeNetwork;
     }
-    if (options?.runId) {
-      this.runId = options.runId;
-      process.env[ServiceEnvironmentVariable.PLAYWRIGHT_SERVICE_RUN_ID] = this.runId;
+    if (!process.env[InternalEnvironmentVariables.MPT_SERVICE_RUN_ID]) {
```

Comment:
Add tests around the same. Also, afaik, existing tests also needs to be modified

Reply:
yes yes I thought once you validate the logic, I will update tests.